src/main/java/com/exasol/containers/VersionBasedExasolDockerImageReference.java

```diff
@@ -16,11 +16,13 @@
 class VersionBasedExasolDockerImageReference implements ExasolDockerImageReference {
     private static final int DEFAULT_DOCKER_IMAGE_REVISION = 1;
     public static final int VERSION_NOT_PRESENT = -1;
+    public static final String PREFIX_NOT_PRESENT = null;
```

Comment:
Actually I'd prefer names like `NULL_PREFIX` and `UNKNOWN_VERSION` as `_NOT_PRESENT` sounds more like a boolean value. 
But as these identifiers are public, we maybe do not want to change them, right?